apps/storybook/.storybook/preview.jsx

```diff
@@ -8,6 +8,11 @@ export const decorators = [
   ),
 ];
 
-export const parameters = {
-  actions: { argTypesRegex: '^on[A-Z].*' },
+const parameters = {
+  decorators: decorators,
+  parameters: {
+    actions: { argTypesRegex: '^on[A-Z].*' },
+  },
 };
+
+export default parameters;
```

Comment:
We tend to avoid default exports to enforce naming during imports - but if this is the only thing it exports its OK

Reply:
Yeah @dmorton-p44 i agree named import is nice thing to have, but with new version storybook expects these to be default exported.
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#70-breaking-changes.